content/docs/started/workstation/appliances/_index.md

```diff
@@ -0,0 +1,5 @@
++++
+title = "Workstation Appliances"
+description = "Detailed instructions for leveraging VM appliances for Kubeflow"
```

Comment:
s/leveraging/using

Reply:
have applied the change, and will move this directory to top level reference section